### PR TITLE
#1834 Fixed Member/Profile types draft status

### LINKED ISSUE #1834: Member/Profile types draft status
Describe the bug
If profile or group type/s are added in 'draft' status they are still visible and able for user to select from.

To Reproduce

Add group type
Change status to draft
Add/Edit group and try to add/change group type
See that draft type/s are still available as option
Expected behavior
Draft types should be not selectable

Screenshots
https://drive.google.com/file/d/1PfS7JMlktvkntLftYj7u_Bboa5s1KhQE/view

Support ticket links
 #https://github.com/buddyboss/buddyboss-theme/issues/1175

### PR DESCRIPTION
**Pull request by dkvasani**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #1834
Fixes #1175

### How to test the changes in this Pull Request:

1. Add group type
2. Change status to draft
3. Add/Edit group and try to add/change group type
4. See that draft type/s are still available as option

Expected behavior
Draft types should be not selectable

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-groups/bp-groups-functions.php [hunk 1]
```diff
@@ -3545,6 +3545,7 @@ function bp_get_active_group_types( $args = array() ) {
 	$args = bp_parse_args( $args, array(
 		'posts_per_page' => - 1,
 		'post_type'      => bp_groups_get_group_type_post_type(),
+                'post_status'    => 'publish',
```
Comment (chetansatasiya) [inline src/bp-groups/bp-groups-functions.php:3548]: @dkvasani Make the tabbing proper use the PHPCBF to make this proper.

File: src/bp-members/bp-members-functions.php [hunk 2]
```diff
@@ -3350,6 +3350,7 @@ function bp_get_active_member_types( $args = array() ) {
 	$args = bp_parse_args( $args, array(
 		'posts_per_page' => - 1,
 		'post_type'      => bp_get_member_type_post_type(),
+                'post_status'    => 'publish',
```
Comment (chetansatasiya) [inline src/bp-members/bp-members-functions.php:3353]: @dkvasani Make the tabbing proper use the PHPCBF to make this proper.

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
@dkvasani Changes looks good. Make the spaces correct in both places.

File: src/bp-groups/bp-groups-functions.php (hunk 1 above)
Comment (dkvasani) [reply src/bp-groups/bp-groups-functions.php]: made tabbing proper

**Review by chetansatasiya — APPROVED** [review]
@jitendrabanjara1991 LGTM 👍 

Merge in dev and add in testing.

**Comment by jitendrabanjara1991** [comment]
Hello @chetansatasiya 
Merged in dev and added a label for testing. thanks

**Comment by mohit-buddyboss** [comment]
@jitendrabanjara1991 

FIXED> Tested on the dev branch of the platform.  Draft Profile and Group Types appearing under drop down list on edit user and edit group page is resolve now and drafted profile and group types are not appearing in the list. 

Video: https://www.loom.com/share/cc07dba0dcd34403a98af1415c5111c3

cc @chetansatasiya

**Pull request merged**